test/Mono.Linker.Tests.Cases/DataFlow/MethodByRefReturnDataFlow.cs

```diff
@@ -30,6 +30,7 @@ public static void Main ()
 		[return: DynamicallyAccessedMembers (DynamicallyAccessedMemberTypes.PublicMethods)]
 		static ref Type ReturnAnnotatedTypeReferenceAsAnnotated () { return ref _annotatedField; }
 
+		[ExpectedWarning ("IL2026", "Message for --TestType.Requires--", ProducedBy = ProducedBy.Trimmer)]
```

Comment:
Ideally all warnings which are only produced by trimmer should have a reason:
* It's by design that analyzer doesn't produce them (something analyzer can't ever do)
* It's a bug/TODO in the analyzer - this should have an issue link so that it's easy to track